docs/contributing.rst

```diff
@@ -132,6 +132,29 @@ mean that this work should be relatively manageable.
 We hope that the lessons we learn building something useful in the short term will be
 applicable in the future as other needs arise.
 
+.. _backwards compatibility with meica:
+
+Is ``tedana`` backwards compatible with MEICA?
+``````````````````````````````````````````````
+
+The short answer is no.
+
+There are two main reasons why. One is that the tool originally used to run the independent
```

Comment:
Can we put each sentence on its own line ?

Reply:
Yep! 🙌